requirements.txt

```diff
@@ -5,7 +5,7 @@ flask-restful==0.3.9
 flask_cors==3.0.10
 pytest
 environs
-pymchelper>=1.10.0
+pymchelper>=2.4.0
```

Comment:
```suggestion
pymchelper==2.4.1
```

Reply:
This one is merged: https://github.com/DataMedSci/pymchelper/pull/631, new version on the way